models/Products/product.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+const uuid = require('uuid/v4');
+const DataModel = require('../file-data-model.js');
+
+class Products extends DataModel {
+  constructor() {
+    super();
+    console.log()
+    this.schema = {
+      category_id: {type: String, required: true},
+      price: { type: Number, required: true },
+      weight: {type: Number},
+      quantity_in_stock: {type: Number,required: true}
+    };
```

Comment:
from line 11-15 you can put this part outside of the class and call it schema. In the parentheses put schema in there.